tests/test_wgpu_native_query_set.py

```diff
@@ -0,0 +1,149 @@
+import wgpu.utils
+
+from testutils import run_tests
+
+
+def test_query_set():
```

Comment:
I think this needs `@mark.skipif(not can_use_wgpu_lib, reason="Needs wgpu lib")`?

Reply:
Sorry, I missed that. Just added.